R/check-class.R

```diff
@@ -227,6 +227,12 @@ check_cran_summary <- function(self, private) {
 
   x <- private$status_
 
+  if (any(vapply(x, function(xx) xx$status, "") %in% 
+     c("in-progress", "created"))) {
+    stop("At least one of the builds has not finished yet. Please wait before calling cran_summary() again.", 
```

Comment:
can you pls break the long line?

Reply:
Also, can you use `map_chr()` Thanks

Reply:
and quote the function name in backticks? Thanks.

Reply:
ok!